src/common.rs

```diff
@@ -343,6 +374,19 @@ impl Info {
     }
 }
 
+impl Bpp {
+    pub(crate) fn into_usize(self) -> usize {
```

Comment:
If I remember correctly, can't you just do `self as usize`?

Reply:
Yes and no. `self as u8` works as well for any enum but isn't always what you meant and will silently wrap. This is just slightly more explicit and affirms that the value range is surely right for the cast.